modules/ROOT/pages/keycloak/coding.adoc

```diff
@@ -90,4 +139,57 @@ try! AgsAuth.instance.configure(authConfig: authenticationConfig)
 .Cordova
 ****
 
-****
\ No newline at end of file
+Import and initialize Auth:
```

Comment:
_... initialize Auth_ is what the section is to describe (it's called initializing the sdk), what about _Instantiate `Auth` and call init method_?